src/pages/ResetPassword/index.js

```diff
@@ -0,0 +1,64 @@
+import { zodResolver } from '@hookform/resolvers/zod';
+import Button from 'components/common/Button';
+import Input from 'components/form/Input';
+import LandingScreen from 'components/landing/LandingScreen';
+import Hamburger from 'components/navigation/Hamburger';
+import useTranslation from 'hooks/useTranslation';
+import { useForm } from 'react-hook-form';
+import { useDispatch } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { error } from 'react-toastify-redux';
+import routesPaths from 'routes/routesPaths';
+import { z } from 'zod';
+import './styles.scss';
+
+const ResetPassword = () => {
+  const t = useTranslation();
+  const dispatch = useDispatch();
+
+  const schema = z.object({
+    email: z.string().email({ message: t('login.errors.emailMsg') }),
+    redirect_url: z.string().default('http://www.example.com'),
+  });
+
+  const {
+    register,
+    formState: { errors },
+  } = useForm({ resolver: zodResolver(schema) });
+
+  const onSubmit = e => {
+    e.preventDefault();
```

Comment:
Are you sure this preventDefault is needed? I think useForm already has it's own.

Reply:
In this case it is because I'm not currently using useForm for password resetting (since AFAIK the current API doesn't support the required flow of sending the email and changing the password while logged out), so this feature is only partially complete.